server/db/mysql/adapter.go

```diff
@@ -57,6 +62,25 @@ type configType struct {
 	MaxOpenConns           int `json:"max_open_conns,omitempty"`
 	MaxIdleConns           int `json:"max_idle_conns,omitempty"`
 	ConnMaxLifetimeSeconds int `json:"conn_max_lifetime_seconds,omitempty"`
+
+	// Single query and transaction timeouts in seconds.
+	// If 0 (or negative), no timeout is applied.
+	SqlTimeoutSeconds int `json:"sql_timeout_seconds,omitempty"`
```

Comment:
The `_seconds` in `sql_timeout_seconds` is good and informative but inconsistent with how it was done before. It implies it was not seconds elsewhere or possibly creates questions. Maybe drop the `_seconds` from the variable name and leave it in comments only?

Reply:
Done.